source/dom/dispatcher.js

```diff
@@ -54,8 +54,20 @@
 		/**
 		* @private
 		*/
+		platformSpecific: function() {
```

Comment:
Or other way is we get window.focus/blur event here. But, not handling on spotlight when it is running on other than webOS platform. @graynorton how do you think?